src/responsive-layout.tsx

```diff
@@ -60,26 +15,19 @@ type Props = {
 };
 
 const ResponsiveLayout: React.FC<Props> = ({children, className, fullWidth, dataAttributes}) => {
-    const classes = useStyles();
     const {isMobile, isTablet} = useScreenSize();
 
-    const sideMargin = isMobile ? MOBILE_SIDE_MARGIN : isTablet ? TABLET_SIDE_MARGIN : null;
-
     const containerType = isMobile ? 'mobile-column' : isTablet ? 'tablet-column' : 'desktop-wide-column';
 
     return (
-        <ResponsiveLayoutMarginContext.Provider value={sideMargin}>
-            <ContainerTypeContext.Provider value={containerType}>
-                <div
-                    className={classnames(classes.container, className)}
-                    {...getPrefixedDataAttributes(dataAttributes)}
-                >
-                    <div className={classnames(classes.layout, {[classes.layoutFullWidth]: fullWidth})}>
-                        {children}
-                    </div>
-                </div>
-            </ContainerTypeContext.Provider>
-        </ResponsiveLayoutMarginContext.Provider>
+        <ContainerTypeContext.Provider value={containerType}>
+            <div
+                className={classnames(styles.container, className)}
+                {...getPrefixedDataAttributes(dataAttributes)}
+            >
+                <div className={fullWidth ? styles.fullWidth : styles.responsiveLayout}>{children}</div>
```

Comment:
previously, fullWidth also adds these

```

        paddingLeft: 'env(safe-area-inset-left)',
        paddingRight: 'env(safe-area-inset-right)',
       width: '100%',
        margin: 0,

        [theme.mq.largeDesktop]: {
            width: 'auto',
            margin: 'auto',
        },
```


none of these are needed now in fullWidth?

Reply:
yes, the safe-area-inset is needed, added. Thanks ;)